src/imgui/ImGuiMachine.cc

```diff
@@ -610,6 +684,13 @@ void ImGuiMachine::paint(MSXMotherBoard* motherBoard)
 	if (showTestHardware) {
 		paintTestHardware();
 	}
+	if (showQuickSetupEditor) {
+		ImGui::PushItemFlag(ImGuiItemFlags_AutoClosePopups, false);
```

Comment:
This `ImGui::{Push,Pop}ItemFlag()` is a leftover from the (failed) popup experiment?